Index.html

```diff
@@ -9,6 +9,13 @@
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Inter">
 </head>
 <body>
+    <header>
+        <ul id="navbar" type="none">
+            <li><a class="nav-icons" href="#">Portfolio</a></li>
+            <li><a class="nav-icons" href="#">About</a></li>
+            <li><a class="nav-icons" href="#">Contact</a></li>
+        </ul>
+    </header>
     <section class="toolbar">
         <p id="logo-text">Welcome</p>
         <div class="buttons" id="menu">
```

Comment:
- [ ] Please use an indentation of two spaces in all your documents. This is a convention and also [best practice.](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

Reply:
Thank you. I have fixed the indentation now.